Package: Create new TOS package (#13763)
Automattic/jetpack: packages/tos/README.md

> # Jetpack Term of Service 

Reviewer: ```suggestion
# Jetpack Terms of Service 
```

A Terms of Service Package that lets us know when a site has agreed to the terms of service.

### Usage

Site agrees to the terms of service.

```php

> use Automattic\Jetpack\TermOfService;

Reviewer: ```suggestion
use Automattic\Jetpack\TermsOfService;
```

> $terms_of_service = new TermOfService();

Reviewer: ```suggestion
$terms_of_service = new TermsOfService();
```

$terms_of_service->agree();
```

> Does the site have the terms of service?

Reviewer: Let's make this clearer, like for example:

```suggestion
Has the master user agreed to the the terms of service for this site?
```

```php

> use Automattic\Jetpack\TermOfService;

Reviewer: ```suggestion
use Automattic\Jetpack\TermsOfService;
```

> $terms_of_service = new TermOfService();

Reviewer: ```suggestion
$terms_of_service = new TermsOfService();
```
